src/sentry/models/releasethreshold.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import FlexibleForeignKey, Model, region_silo_only_model
+
+
+@region_silo_only_model
+class ReleaseThreshold(Model):
+    __relocation_scope__ = RelocationScope.Excluded
+
+    class ReleaseThresholdType(models.TextChoices):
+        TOTAL_ERROR_COUNT = "TEC", _("Total Error Count")
+        NEW_ISSUE_COUNT = "NIC", _("New Issue Count")
+        UNHANDLED_ISSUE_COUNT = "UIC", _("Unhandled Issue Count")
+        REGRESSED_ISSUE_COUNT = "RIC", _("Regressed Issue Count")
+        FAILURE_RATE = "FR", _("Failure Rate")
+        CRASH_FREE_SESSION_RATE = "CFSR", _("Crash Free Session Rate")
+        CRASH_FREE_USER_RATE = "CFUR", _("Crash Free User Rate")
+
+    class ValueType(models.TextChoices):
+        PERCENT_OVER = "PO", _("Percent Over")
+        PERCENT_UNDER = "PU", _("Percent Under")
+        ABSOLUTE_OVER = "AO", _("Absolute Over")
+        ABSOLUTE_UNDER = "AU", _("Absolute Under")
+
+    threshold_type = models.CharField(choices=ReleaseThresholdType.choices, max_length=120)
+    value_type = models.CharField(choices=ValueType.choices, max_length=120)
```

Comment:
hmmm I wonder if "value" is correct here... maybe "validation_method"?

Reply:
this includes over/under AND absolute/pecentage, I figured these were captured well with this name, `validation_method` sounds off to me, should I break this into two variables? maybe value type (which is absolute or percentage) _and_ `validation_method` which is over/under?

Reply:
mmm i think splitting it would be overkill.

We just need to find a better name... 'value' feels off since over/under isn't really a value, neither is absolute/percent 🤔 

these are the comparison triggers maybe?
Maybe 'trigger_type'? (i think i like this the most)

Reply:
Also just thought of another possible trigger type

Lets say we're looking at an uptime %
We want to make sure the % _over the whole window_ is above X%... Not quite sure what this would be called... like.. It's a window_percent maybe? 🤷